ui/packages/form/src/Dropdown.svelte

```diff
@@ -1,67 +1,356 @@
 <script lang="ts">
```

Comment:
This file is getting too large. It needs to be broken down into smaller components, so it is easier to reason about.